modules/microshift-install-system-requirements.adoc

```diff
@@ -8,7 +8,8 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A supported version of {op-system} or {op-system-ostree}.
+* A compatible version of {op-system} or {op-system-ostree}.
+* Aarch64 or x86_64 system architecture.
```

Comment:
I more commonly see it called "AArch64," and that's what I found in the [Arm Architecture Reference Manual for A-profile architecture](https://developer.arm.com/documentation/ddi0487/latest), but we're all over the place with it in the docs.

Reply:
@maxwelldb You're right, it's here: https://redhat-documentation.github.io/supplementary-style-guide/#aarch64.